src/scanpy/preprocessing/_pca/__init__.py

```diff
@@ -265,6 +265,17 @@ def pca(
         for chunk, start, end in adata_comp.chunked_X(chunk_size):
             chunk = chunk.toarray() if issparse(chunk) else chunk
             X_pca[start:end] = pca_.transform(chunk)
+    elif is_dask and issparse(X._meta):
+        from ._dask_sparse import PCASparseDask
+
+        if random_state != 0:
+            msg = f"Ignoring {random_state=} when using a sparse dask array"
+            warnings.warn(msg)
+        if svd_solver not in {None, "arpack", "auto"}:
```

Comment:
Can we make `svd_solver` to be `None` as the "our covariance implementation when dask+sparse" and the rest can be ignored for now, and then error later?

Reply:
I don’t quite get that, can you elaborate please?

Reply:
OK, added `'covariance_eigh'` option.